src/system/permissions-user-roles.md

```diff
@@ -31,14 +31,14 @@ To give someone restricted access to the Admin, the first step is to create a ro
     - `All`
     - `Custom`
 
-    If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
+    {:.ee-only} If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```

Comment:
That tag works on line items but not paragraphs (something we should add to the theme). In this case, we will have to use a span:

```suggestion
<span class="ee-only"></span>If set to `Custom` for a multisite installation, select the checkbox of the website and store where the role is to be used.
```